### PR TITLE
increase max-jobs to 64

### PR DESCRIPTION
**Pull request by figsoda**
copied from https://github.com/nix-community/aarch64-build-box/blob/40046e78a85664e1ea050af5a2799e3f4fd9269a/configuration.nix#L73-L74

### CODE REVIEW

**Comment by winterqt** [comment]
Have you ever run into a situation where the limit is an issue? I suppose it shouldn't matter too much, since in the worst case scenario we lower it, but just wanna make sure this is needed :)

**Comment by figsoda** [comment]
I do find building multiple derivations significantly slower on darwin-build-box compared to aarch64-build-box, so I thought that might be the issue

btw I didn't test this since I don't know how to

**Comment by winterqt** [comment]
> I do find building multiple derivations significantly slower on darwin-build-box compared to aarch64-build-box

The Linux box always builds slowly for me :(

**Review by winterqt — APPROVED** [review]
Let's try it.

File: darwin-configuration.nix [hunk 1]
```diff
@@ -16,6 +16,9 @@
   nix.settings.sandbox = "relaxed";
   nix.settings.extra-platforms = [ "x86_64-darwin" ];
 
+  nix.nrBuildUsers = config.nix.settings.max-jobs * 2;
```
Comment (winterqt) [inline darwin-configuration.nix:19]: x2 for extra headroom? Would it ever actually be used? (Fine either way, just curious.)

Comment (figsoda) [reply darwin-configuration.nix:19]: just copied over since that's what aarch64-build-box is using, not sure if we need this

Comment (winterqt) [reply darwin-configuration.nix:19]: If Graham did it, it's probably good, assuming it even evals.

**Pull request merged**